OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

```diff
@@ -177,6 +176,33 @@ public void TestGetOptimizelyConfigSDKAndEnvironmentKeyDefault()
             Assert.AreEqual(optimizelyConfig.EnvironmentKey, "");
         }
 
+        [Test]
+        public void TestGetOptimizelyConfigWithOdpIntegration()
+        {
+            var datafileProjectConfig = DatafileProjectConfig.Create(TestData.TypedAudienceDatafile, new NoOpLogger(), new ErrorHandler.NoOpErrorHandler());
```

Comment:
I think we need a separate datafile for integration testing instead of reusing "TypedAudienceDatafile".